src/search/search-index-new/search/index.ts

```diff
@@ -45,6 +47,7 @@ export async function search({
         }
     }
 
+    // WTF
```

Comment:
Totally agree with this "WTF" 😅 hopefully we can clean a lot of this weird stuff once we have time to get rid of compatibility with the old index stuff (once all that migration stuff we've been talking about the last few months is confirmed to be over)